### ISSUE TITLE
 Add Vaccine Related Info: APIs available now

### ISSUE THREAD

**Issue by MadHatter92** [coronasafe/life#182]
https://apisetu.gov.in/public/marketplace/api/cowin

**Comment by rawris** [comment]
@MadHatter92 Any idea if we can also book appointments using any cowin api?

**Comment by MadHatter92** [comment]
No, only check appointment and download certificate https://apisetu.gov.in/public/api/cowin

**Comment by tejasa97** [comment]
Yes in fact I believe appointments can be scheduled using these APIs:
[https://apisetu.gov.in/public/marketplace/api/cowin/cowin-protected-v2](url)
